src/base/name/builder.rs

```diff
@@ -266,6 +266,70 @@ where
         Ok(())
     }
 
+    /// Appends a label with the decimal representation of `u8`.
```

Comment:
I don't understand how this is "decimal" representation, it looks like a conversion to ASCII, what is "decimal" a reference to here? Also, a unit test might be good to have for this function.

Reply:
Decimal as in “representation of the value using decimal digits” – I think that’s sort of clear?

Reply:
Is not "Appends a label with the decimal interpretation of `u8`" rather than representation?

Reply:
Strictly speaking, we are not interpreting the value (i.e., trying to determine its meaning) but only represent it as text.

Reply:
I read the arithmetic you are doing as interpreting the value. You are assuming its format and interpreting it according to the rules of that format. Anyway, if you feel that the name is correct please ignore this and move on.

Reply:
I think decimal representation is the canonical expression in this case. https://en.wikipedia.org/wiki/Decimal_representation